src/util/get-github-client-config.ts

```diff
@@ -57,9 +57,11 @@ async function calculateProxyBaseUrl(jiraHost: string, gitHubBaseUrl: string | u
 			skipOutboundProxy = false;
 		}
 		if (skipOutboundProxy) {
+			logger.warn("Skip outbound proxy");
```

Comment:
Does these two lines include the jira host and ghe server url as well? 
If not, how can these two logging lines help us?

Reply:
Yes, logger is a child of the request's logger, and it has all the data. Not sure about GHE server URL, but if it is logged on error, this line can be found using request_Id